pkgs/development/interpreters/python/hooks/fix-python-deps.sh

```diff
@@ -0,0 +1,76 @@
+# shellcheck shell=bash
+
+# Setup hook that fixes Python dependencies versions.
+#
+# Example usage in a derivation:
+#
+#   { …, pythonPackages, … }:
+#
+#   pythonPackages.buildPythonPackage {
+#     …
+#     nativeBuildInputs = [ pythonPackages.fixPythonDepsHook ];
+#
+#     # Already the default, only change if necessary
+#     requirementsTxt = "requirements.txt";
```

Comment:
I don't think this currently would fail if I provided a none existing file here.

Reply:
Yeah it wouldn't. This was just an example if the file has another name.

Probably need to change the example to be more clear, though.